server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/data/DataType.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.index.compositeindex.datacube.startree.data;
+
+/**
+ * Data type of doc values
+ * @opensearch.internal
+ */
+public enum DataType {
+    INT(Integer.BYTES, true),
+    LONG(Long.BYTES, true),
+    FLOAT(Float.BYTES, true),
+    DOUBLE(Double.BYTES, true);
+
+    private final int size;
+    private final boolean numeric;
+
+    DataType(int size, boolean numeric) {
+        this.size = size;
+        this.numeric = numeric;
+    }
+
+    /**
+     * Returns the number of bytes needed to store the data type.
+     */
+    public int size() {
```

Comment:
Lets remove methods in this class which is not used.

Reply:
ack!